packaging/container-deployment/Containerfile.bytecode

```diff
@@ -0,0 +1,14 @@
+FROM scratch
+
+ARG PROGRAM_NAME
+ARG SECTION_NAME
+ARG PROGRAM_TYPE
+ARG BYTECODE_FILENAME
+ARG KERNEL_COMPILE_VER
+
+COPY  $BYTECODE_FILENAME /etc/bpfd/bytecode/
```

Comment:
being in the root is probably more convenient - both for us and users.
`COPY $BYTECODE_FILENAME $BYTECODE_FILENAME`

Reply:
Done